tests/tests_integration/test_api/test_data_workflows.py

```diff
@@ -540,3 +541,33 @@ def test_trigger_run_history(
         )
         # it would take too long to wait for the trigger to run, so we just check that the history is not None
         assert history is not None
+
+
+class TestSimIntInWorkflows:
+    def test_create_and_run_simint_workflow(
```

Comment:
Lets move this test into `TestWorkflowVersions`. In addition, use the try-finally pattern and a random string in the workflow id. See the `test_upsert_delete` test in that class. 

Note that the test name is misleading as you are not running the workflow. Suggest renaming to 
`test_upsert_delete_with_simulation_task`